corehq/apps/hqwebapp/templates/hqwebapp/includes/bootstrap3/modal_solutions_feature_request.html

```diff
@@ -0,0 +1,110 @@
+                        </div>
+                        <div class="form-group">
+                            <label for="request-report-message"
+                                   class="col-sm-3 control-label">{% trans "Full Description" %}</label>
+                            <div class="col-sm-9">
+                                <textarea name="message"
+                                          class="form-control vertical-resize"
+                                          id="request-report-message"
+                                          rows="3"
+                                          placeholder="{% trans "Please describe the product changes you would like to see in CommCare." %}"></textarea>
+                            </div>
+                        </div>
+                        <div class="form-group" id="request-report-cc-form-group">
+                            <label for="request-report-cc"
+                                   class="col-sm-3 control-label">{% trans "Other recipients" %}</label>
+                            <div class="col-sm-9">
+                                <input type="text"
+                                       name="cc"
+                                       class="form-control"
+                                       id="request-report-cc"
+                                       placeholder="ex: admin@commcarehq.org, test@commcarehq.org" />
+                                <p class="help-block">
+                                    {% blocktrans %}
+                                        Comma-separated email addresses of others you want to notify about this request.
+                                    {% endblocktrans %}
+                                    <span class="label label-danger hide">
+                                        <i class="fa fa-warning"></i> {% trans 'Incorrect Format' %}
+                                    </span>
+                                </p>
+                            </div>
+                        </div>
+                        <div class="form-group">
+                            <label for="request-report-file"
+                                   class="col-sm-3 control-label">{% trans 'Upload file' %}</label>
+                            <div class="col-sm-9">
+                                <input type="file" id="request-report-file" name="feature_request"/>
+                                <p class="help-block">
+                                    {% blocktrans %}
+                                    Please upload any files relevant to your suggestion.
+                                    {% endblocktrans %}
+                                </p>
+                                <p class="help-block">
+                                    <i class="fa fa-warning"></i>
+                                    {% blocktrans %}
+                                        Do not upload a file containing individually identifiable data.
```

Comment:
nit: how about "file containing sensitive data like passwords or project data"

I guess its important to be explicit to user what should be avoided.

Reply:
Sounds good, it would make it a bit more clear to the user. Since I mostly copied this template from the bug report, I suppose we should change the text there as well to be more descriptive?

Reply:
Good to suggest that to SaaS first. I don't think we should simply change that modal though this one should be safe to update.

Reply:
Good point. Given that, I'll leave the bug report text for now as it's out of scope for this PR.